### ISSUE TITLE
Fine-tune using x2 scale

### ISSUE THREAD

**Issue by dalia-jaber** [xinntao/Real-ESRGAN#145]
Hello, can you please provide the following model **RealESRNet_x2plus.pth** ? What are the models used to fine-tune the super-resolution model x2?

**Comment by KleinXin** [comment]
same question

**Comment by shakibakh** [comment]
same question

**Comment by HosseinMoazzam** [comment]
The same question.

**Comment by HosseinMoazzam** [comment]
I made some changes to the finetune_realesrgan_x4plus.yml file and used the RealESRGAN_x2plus.pth model instead of RealESRNet_x2plus.pth for fine-tuning the x2 model. Here is my .yml file for this work. 

[finetune_realesrgan_x2plus.zip](https://github.com/xinntao/Real-ESRGAN/files/13043618/finetune_realesrgan_x2plus.zip)

**Comment by wangxinchao-bit** [comment]
have you solved the problem？

**Comment by HosseinMoazzam** [comment]
> have you solved the problem？

Yes, I solved the problem by the last comment that I wrote above.

**Comment by 201210302** [comment]
@xinntao the same question

**Comment by xiaotian3** [comment]
> I made some changes to the finetune_realesrgan_x4plus.yml file and used the RealESRGAN_x2plus.pth model instead of RealESRNet_x2plus.pth for fine-tuning the x2 model. Here is my .yml file for this work.
> 
> [finetune_realesrgan_x2plus.zip](https://github.com/xinntao/Real-ESRGAN/files/13043618/finetune_realesrgan_x2plus.zip)

it solved the x2 fine-tune!
